governance/multisig_wh_message_builder/tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
   "compilerOptions": {
     "declaration": true,
+    "composite": true,
+    "declarationMap": true,
+    "incremental": true,
```

Comment:
These three configs will make typescript incremental and enable caching for that. and declarationMap will create a mapping between declarations and source that will make go to definitions in VS Code work (for local dependencies)

As a side-effect of `composite` I needed to set `rootDir` explicitly.